### DISCUSSION TITLE
School somewere

### DISCUSSION THREAD

**Discussion by ger1964jou** [community/community#117505] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Éducation sans abus juste aimance a la vie de ses enfants et parentales qu’elles que se soient de nous tous.

**Comment by ger1964jou** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> Éducation sans abus juste aimance a la vie de ses enfants et parentales qu’elles que se soient de nous tous.

**Comment by MIRELOSZKA196** [comment]
G
